autofaiss/external/build.py

```diff
@@ -108,7 +108,8 @@ def create_index(
     distributed: Optional[str] = None,
     temporary_indices_folder: str = "hdfs://root/tmp/distributed_autofaiss_indices",
     file_counts: List[int] = None,
-):
+    nb_indices_to_keep: int = 1,
```

Comment:
this value should be used to adapt the estimation of the memory required to build the index

Reply:
Yes, indeed.
I have implemented as follows:
```
if nb_indices_to_keep > 1:
        if distributed is None:
            logger.error('nb_indices_to_keep can only be larger than 1 when distributed is "pyspark"')
            return None, None
        current_memory_available = (
            f"{float(current_memory_available[:-1]) * nb_indices_to_keep}{current_memory_available[-1]}"
        )
```